src/Features/WetnessEffects.cpp

```diff
@@ -192,161 +151,154 @@ void WetnessEffects::DrawSettings()
 
 	ImGui::Spacing();
 	ImGui::Spacing();
-
-	if (ImGui::TreeNodeEx("Statistics", ImGuiTreeNodeFlags_DefaultOpen)) {
-		ImGui::Text(std::format("Wetness depth : {:.2f}", wetnessDepth / WETNESS_SCALE).c_str());
-		ImGui::Text(std::format("Puddle depth : {:.2f}", puddleDepth / PUDDLE_SCALE).c_str());
-		ImGui::Spacing();
-		ImGui::Spacing();
-		ImGui::Text(std::format("Current weather : {0:X}", currentWeatherID).c_str());
-		ImGui::Text(std::format("Previous weather : {0:X}", lastWeatherID).c_str());
-		ImGui::TreePop();
-	}
 }
 
-float WetnessEffects::CalculateWeatherTransitionPercentage(float skyCurrentWeatherPct, float beginFade, bool fadeIn)
+class BGSShaderParticleGeometryDataVR : public RE::TESForm
```

Comment:
use my PR for improved RE for this https://github.com/doodlum/skyrim-community-shaders/pull/757